Fix broken TASOC tests

<!-- Fill in the information below before opening an issue. -->

#### Problem description
<!-- Provide a clear and concise description of the issue. -->

Tests with `--remote-data` enabled are consistently failing (example [CI run](https://github.com/lightkurve/lightkurve/runs/6066111149?check_suite_focus=true)). It seems to be because new [DR5 release](https://archive.stsci.edu/hlsp/tasoc) replaces the DR4 data, breaking some of the tests. Affected tests:

```
FAILED tests/test_units.py::test_tasoc_ppm_units - TypeError: 'NoneType' obje...
FAILED tests/io/test_tasoc.py::test_detect_tasoc - urllib.error.HTTPError: HT...
FAILED tests/io/test_tasoc.py::test_read_tasoc - urllib.error.HTTPError: HTTP...
FAILED tests/io/test_tasoc.py::test_search_tasoc - assert 0 == 1
```

- `test_detect_tasoc() / test_read_tasoc()`: the data product has a new URL:
https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/tasoc/s0001/c1800/0000/0004/1206/4070/hlsp_tasoc_tess_ffi_tic00412064070-s0001-cam3-ccd1-c1800_tess_v05_ens-lc.fits

- `test_search_tasoc()`: the given TIC / sector is still in DR5 release according to their [bulk  download script](https://archive.stsci.edu/hlsps/tasoc/download_scripts/hlsp_tasoc_tess_ffi-tpf_s0001_tess_v05_lc-download.sh). So it might be an interim problem that MAST database has not yet been updated.
- `test_tasoc_ppm_units()`: similar problem (the `HV 2112` referenced is [TIC 183306762](https://exofop.ipac.caltech.edu/tess/target.php?id=183306762), which does exists in the bulk download script)

--- a/tests/io/test_tasoc.py
+++ b/tests/io/test_tasoc.py
@@ -9,10 +9,15 @@ from lightkurve.io.tasoc import read_tasoc_lightcurve
 from lightkurve.io.detect import detect_filetype
 
 
+# The URL needs to be updated upon a new TASOC data release.
+TEST_TIC_ID = 150441810
+TEST_FIT_URL = "https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/tasoc/s0001/c1800/0000/0001/5044/1810/hlsp_tasoc_tess_ffi_tic00150441810-s0001-cam4-ccd4-c1800_tess_v05_ens-lc.fits"
+
+
 @pytest.mark.remote_data
 def test_detect_tasoc():
     """Can we detect the correct format for TASOC files?"""
-    url = "https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/tasoc/s0001/ffi/0000/0004/1206/4070/hlsp_tasoc_tess_ffi_tic00412064070-s01-c1800_tess_v04_lc.fits"
+    url = TEST_FIT_URL
     f = fits.open(url)
 
     assert detect_filetype(f) == "TASOC"
@@ -21,7 +26,7 @@ def test_detect_tasoc():
 @pytest.mark.remote_data
 def test_read_tasoc():
     """Can we read TASOC files?"""
-    url = "https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/tasoc/s0001/ffi/0000/0004/1206/4070/hlsp_tasoc_tess_ffi_tic00412064070-s01-c1800_tess_v04_lc.fits"
+    url = TEST_FIT_URL
     with fits.open(url, mode="readonly") as hdulist:
         fluxes = hdulist[1].data["FLUX_RAW"]
 
@@ -34,9 +39,8 @@ def test_read_tasoc():
 @pytest.mark.remote_data
 def test_search_tasoc():
     """Can we search and download a TASOC light curve?"""
-    search = search_lightcurve("TIC 412064070", author="TASOC", sector=1)
-    assert len(search) == 1
+    search = search_lightcurve(f"TIC {TEST_TIC_ID}", author="TASOC")
+    assert len(search) >= 1
     assert search.table["author"][0] == "TASOC"
     lc = search.download()
     assert type(lc).__name__ == "TessLightCurve"
-    assert lc.sector == 1

--- a/tests/test_units.py
+++ b/tests/test_units.py
@@ -3,6 +3,7 @@ import pytest
 import lightkurve as lk  # necessary to enable the units tested below
 from astropy import units as u
 
+from .io.test_tasoc import TEST_TIC_ID
 
 def test_custom_units():
     """Are ppt, ppm, and percent enabled AstroPy units?"""
@@ -14,6 +15,7 @@ def test_custom_units():
 @pytest.mark.remote_data
 def test_tasoc_ppm_units():
     """Regression test for #956."""
-    lc = lk.search_lightcurve('HV 2112', author='TASOC', sector=1, exptime=1800).download()
+
+    lc = lk.search_lightcurve(f"TIC {TEST_TIC_ID}", author='TASOC').download()
     assert lc['flux_corr'].unit == "ppm"
     assert "Unrecognized" not in repr(lc['flux_corr'].unit)

